2021-Shenzhen-FinTechathon3/WeHeart/Project/WeHeart/build/tmp/compileJava/source-classes-mapping.txt

```diff
@@ -0,0 +1,379 @@
+org/example/demo/model/bo/ProgramGetnumberInputBO.java
+ org.example.demo.model.bo.ProgramGetnumberInputBO
+org/example/demo/model/bo/OrganizationAddprogramInputBO.java
+ org.example.demo.model.bo.OrganizationAddprogramInputBO
+org/example/demo/model/bo/IntegralAddPointsInputBO.java
+ org.example.demo.model.bo.IntegralAddPointsInputBO
+org/example/demo/service/CharityTableService.java
+ org.example.demo.service.CharityTableService
+org/example/demo/model/bo/IndividualDealPointsInputBO.java
+ org.example.demo.model.bo.IndividualDealPointsInputBO
+org/example/demo/config/ContractConfig.java
+ org.example.demo.config.ContractConfig
+org/example/demo/model/bo/DAGGetparentsInputBO.java
+ org.example.demo.model.bo.DAGGetparentsInputBO
+org/example/demo/model/bo/OrganizationImplementProgramInputBO.java
+ org.example.demo.model.bo.OrganizationImplementProgramInputBO
+org/example/demo/model/bo/IndividualGetprocessInputBO.java
+ org.example.demo.model.bo.IndividualGetprocessInputBO
+org/example/demo/model/bo/OrganizationGetDAGInputBO.java
+ org.example.demo.model.bo.OrganizationGetDAGInputBO
+org/example/demo/model/bo/CallRingSigSet_messageInputBO.java
+ org.example.demo.model.bo.CallRingSigSet_messageInputBO
+org/example/demo/model/bo/DAGGetblocknumberInputBO.java
+ org.example.demo.model.bo.DAGGetblocknumberInputBO
+org/example/demo/raw/Table.java
+ Table
+ Table$1
+ Table$10
+ Table$11
+ Table$12
+ Table$13
+ Table$2
+ Table$3
+ Table$4
+ Table$5
+ Table$6
+ Table$7
+ Table$8
+ Table$9
+org/example/demo/config/SdkBeanConfig.java
+ org.example.demo.config.SdkBeanConfig
+org/example/demo/model/bo/CallRingSigRing_sigInputBO.java
+ org.example.demo.model.bo.CallRingSigRing_sigInputBO
+org/example/demo/utils/IOUtil.java
+ org.example.demo.utils.IOUtil
+org/example/demo/model/bo/IndividualGetinforInputBO.java
+ org.example.demo.model.bo.IndividualGetinforInputBO
+org/example/demo/model/bo/ProgramGetinforInputBO.java
+ org.example.demo.model.bo.ProgramGetinforInputBO
+org/example/demo/controller/IndividualController.java
+ org.example.demo.controller.IndividualController
+org/example/demo/model/bo/CharityTableCreateInputBO.java
+ org.example.demo.model.bo.CharityTableCreateInputBO
+org/example/demo/raw/CharityTable.java
+ CharityTable
+ CharityTable$1
+ CharityTable$10
+ CharityTable$11
+ CharityTable$12
+ CharityTable$13
+ CharityTable$14
+ CharityTable$15
+ CharityTable$16
+ CharityTable$17
+ CharityTable$18
+ CharityTable$19
+ CharityTable$2
+ CharityTable$20
+ CharityTable$21
+ CharityTable$22
+ CharityTable$3
+ CharityTable$4
+ CharityTable$5
+ CharityTable$6
+ CharityTable$7
+ CharityTable$8
+ CharityTable$9
+ CharityTable$CreateResultEventResponse
+ CharityTable$InsertResultEventResponse
+ CharityTable$RemoveResultEventResponse
+ CharityTable$UpdateResultEventResponse
+org/example/demo/model/bo/IndividualChangevalueInputBO.java
+ org.example.demo.model.bo.IndividualChangevalueInputBO
+org/example/demo/service/IntegralService.java
```

Comment:
> __[Secret Scan]__ It may be dangerous to commit the AWS secret access key. ([view](https://sider.review/gh/repos/195968114/pulls/54#issue-6236415))
>
> <details>
>
> AWS secret access key is an important credential that allows you to use AWS programmatically.
> Once this leaks, an attacker can access and operate your AWS resources.
>
> See also:
> * https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html
>
> | Rule | Severity |
> | ---- | -------- |
> | `sider.secrets.aws.secret_access_key` | `warning` |
>
> You can [close this issue](https://sider.review/gh/repos/195968114/pulls/54#issue-6236415) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

this is not a issue, I think.